src/main/webapp/postcard.html

```diff
@@ -25,15 +25,25 @@
       </ul>
     </nav>
 
+    <section class="page-section">
+      <div class="container">
+        <div class="text-center">
+          <h1>Your postcard is ready!</h1>
+          <div class="pcontainer"></div>
+          
+          <h1>Who is the receiver?</h1>
+          <form class="cf">
```

Comment:
change to: 
      <form method="POST" action="/mail" id="input_email">
        <input type="text" id="input_text" name="mail" value="email"> 
      </form>
      <button onclick="send()" id="submit_send">SEND</button>
      <div id="sent"></div>

so that it works with the rest of the code